user-docs/pupil-capture.md

```diff
@@ -209,67 +224,190 @@ If you open up a session folder you will see a collection of video(s) and data f
+during this time range as a fixation.
+
+The `Fixation Detector 3D` plugin uses the eye model's 3d orientation angle to
+calculate the dispersion. The settings allow you to set the duration in seconds
+and the maximum dispersion amount in degrees.
+
+If the plugin is loaded, fixations will reduce the amount of samples that a
+[Screen Marker or Manual Marker calibration](https://docs.pupil-labs.com/master/#calibration-methods) usually requires.
+
+### Network plugins
+
+Pupil Capture has a built-in data broadcast functionality. It is based on the network library [ZeroMQ](http://zeromq.org/)
+and follows the [`PUB-SUB` pattern](http://zguide.zeromq.org/php:chapter1#Getting-the-Message-Out). Data is published with an affiliated topic.
+Clients need to subscribe to their topic of interest to receive the respective data. To save network traffic, only data
+with at least one subscription is transferred.
+
+#### Pupil Remote
+
+> {{< video-webm src="/videos/pupil-remote/pr.webm" >}}
+
+`Pupil Remote` is the plugin that functions as entry point to the broadcast infrastructure. It also provides a high level
+interface to control Pupil Capture over the network (e.g. start/stop a recording).
+
+* Load the `Pupil Remote` plugin from the `General` sub-menu in the GUI (it is loaded by default).
+* It will automatically open a network port at the default `Address`.
+* Change the address and port as desired.
+* If you want to change the address, just type in the address after the `tcp://`
+
+<aside class="notice">
+See the developer documentation on how to access Pupil Remote from your own application.
+</aside>
+
+#### Pupil Groups
+
 `Pupil Groups` can help you to collect data from different devices and control an experiment with multiple actors (data generators and sensors) or use more than one Pupil device simultaneously:
 
 * Load the `Pupil Groups` plugin from the `General` sub-menu in the GUI.
-* Once the plugin is active it will show all other local network pupil sync nodes in the GUI
-* It will also automatically synchronise time up to 0.1ms.
+* Once the plugin is active it will show all other local network Pupil Group nodes in the GUI
 * Furthermore actions like starting and stopping a recording on one device will be mirrored instantly on all other devices.
 
+<aside class="notice">
 For this to work your network needs to allow `UDP` transport. If the nodes do not find each other, create a local wifi network and use that instead.
+</aside>
 
 <aside class="notice">
   Pupil Groups can easily be integrated in your own app or device. Have a look at <a href="https://github.com/pupil-labs/pupil-helpers/tree/master/pupil_sync">pupil helpers</a> to get started.
 </aside>
 
-### Streaming Pupil Data over the network
+<!-- ### Streaming Pupil Data over the network
 
-> {{< video-webm src="/videos/pupil-remote/pr.webm" >}}
+Its network interface is based on the ZeroMQ [`REQ-REP` pattern](http://zguide.zeromq.org/php:chapter1#Ask-and-Ye-Shall-Receive).
```

Comment:
Does this not require some sort of context/introduction? This first sentence seems a bit out of place.